cmd/edituser.go

```diff
@@ -439,7 +439,7 @@ func (p *UserPermissionLimits) Run(ctx ActionCtx, claim *jwt.UserPermissionLimit
 
 	if flags.Changed("bearer") {
 		claim.BearerToken = p.bearer
-		r.AddOK("changed bearer to %t", p.bearer)
+		r.AddOK("ensured bearer is %t", p.bearer)
```

Comment:
I wouldn't say "ensured" (I know that is what Phil said). I think the intention should be to say, "changed bearer to true" or if there was no change "ignoring change to bearer - value is already x"

Reply:
OK, I updated the PR to do the following:

```
$ nsc edit user --deny-sub foo --bearer=true
[ OK ] changed bearer to true
[ OK ] added deny sub "foo"
[ OK ] generated user creds file `~/.local/share/nats/nsc/keys/creds/dreamy_driscoll/myacc/myusr.creds`
[ OK ] edited user "myusr"

$ nsc edit user --deny-sub foo --bearer=false
[ OK ] ignoring change to bearer - value is already false
[ OK ] added deny sub "foo"
[ OK ] generated user creds file `~/.local/share/nats/nsc/keys/creds/dreamy_driscoll/myacc/myusr.creds`
[ OK ] edited user "myusr"

$ nsc edit user --deny-sub foo --bearer
[ OK ] changed bearer to true
[ OK ] added deny sub "foo"
[ OK ] generated user creds file `~/.local/share/nats/nsc/keys/creds/dreamy_driscoll/myacc/myusr.creds`
[ OK ] edited user "myusr"

$ nsc edit user --deny-sub foo
[ OK ] added deny sub "foo"
[ OK ] generated user creds file `~/.local/share/nats/nsc/keys/creds/dreamy_driscoll/myacc/myusr.creds`
[ OK ] edited user "myusr"
```